### PR TITLE
[MacOS] Image Aspect Fix

### PR DESCRIPTION
**Pull request by MarkoBL**
[macOS] Image Aspect Ratio Implementation

### Description of Change ###

This PR implements image aspect ratios for macOS. NSImageView doesn't support AspectFill, like iOS. Therefore, I used a NSView with a CALayer to mimic a NSImageView.

### Bugs Fixed ###

- Provide links to bugs here

### Behavioral Changes ###

Before this PR, the macOS platform didn't have image aspect ratios implemented.
### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

**Comment by MarkoBL** [comment]
2nd try for this PR: https://github.com/xamarin/Xamarin.Forms/pull/741

Please let me know what I should change to get this merged. Thanks.

File: Xamarin.Forms.Platform.MacOS/Controls/FormsImageView.cs [hunk 1]
```diff
@@ -1,11 +1,19 @@
-﻿using AppKit;
+﻿using System;
+using AppKit;
+using CoreAnimation;
 
 namespace Xamarin.Forms.Platform.MacOS
 {
-	internal class FormsNSImageView : NSImageView
+	public class FormsNSImageView : NSView
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.MacOS/Controls/FormsImageView.cs]: please keep internal

File: Xamarin.Forms.Platform.MacOS/Renderers/ImageRenderer.cs [hunk 2]
```diff
@@ -1,10 +1,12 @@
 ﻿using System;
 using System.ComponentModel;
 using AppKit;
+using CoreAnimation;
+using CoreGraphics;
 
 namespace Xamarin.Forms.Platform.MacOS
 {
-	public class ImageRenderer : ViewRenderer<Image, NSImageView>
+	public class ImageRenderer : ViewRenderer<Image, FormsNSImageView>
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.MacOS/Renderers/ImageRenderer.cs]: Nah we can't have this

**Comment by MarkoBL** [comment]
I guess you meant NSView. A NSImageView is not required.

**Pull request merged**